src/cli/cli_tcat.cpp

```diff
@@ -241,6 +269,20 @@ template <> otError Tcat::Process<Cmd("devid")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli tcat start
+ * @code
+ * tcat start
+ * Done
+ * @endcode
+ * Starts TCAT operation.
```

Comment:
Need a @par before this line so the command description is pulled into our CLI reference

Reply:
I'm not sure if I got the line correctly please double check.

Reply:
Yeah it should go before the text, not after.  Suggested edits for this and the other.